packages/@aws-cdk/aws-cloudwatch/lib/metric-util.ts

```diff
@@ -0,0 +1,297 @@
+import { Duration } from "@aws-cdk/core";
+import { MathExpression } from "./metric";
+import { IMetric, MetricConfig, MetricExpressionConfig, MetricStatConfig } from "./metric-types";
+
+/**
+ * Return the JSON structure which represents these metrics in a graph.
+ *
+ * Depending on the metric type (stat or expression), one `Metric` object
+ * can render to multiple time series.
+ *
+ * - Top-level metrics will be rendered visibly, additionally added metrics will
+ *   be rendered invisibly.
+ * - IDs used in math expressions need to be either globally unique, or refer to the same
+ *   metric object.
+ *
+ * This will be called by GraphWidget, no need for clients to call this.
+ */
+export function allMetricsGraphJson(left: IMetric[], right: IMetric[]): any[][] {
```

Comment:
The alternative is `unknown[][]` which we're just going to cast away on the other end of this private function.

It's arbitrary structured data. We traditionally represent this as `any`. Please don't start imposing new standards on me right now.

Reply:
I'm not trying to impose new standards. If that's the best that can be, okay. I don't like to see so much `any` because it's option out of many type-checking features and makes it easy to creep bugs in.